internal/lsp/server.go

```diff
@@ -788,10 +832,8 @@ func (s *Server) handleCompletion(ctx context.Context, languageService *ls.Langu
 		params.Position,
 		params.Context,
 		getCompletionClientCapabilities(s.initializeParams),
-		&ls.UserPreferences{
-			IncludeCompletionsForModuleExports:    core.TSTrue,
-			IncludeCompletionsForImportStatements: core.TSTrue,
-		})
+		languageService.UserPreferences(),
```

Comment:
We should probably drop these params, as they are already on `languageService` methods, so the method can already ask for `UserPreferences`.